README.md

```diff
@@ -54,6 +54,50 @@ pyyaml
 urllib3
 ```
 
+### Developer setup
+
+Clone the repository using `git`:
+```
+git clone https://github.com/zowe/zowe-client-python-sdk.git
+```
+
+We recommend that developers create a virtual environment to install all the required dependencies for this project.  
+**Note:** Use of `venv`, the virtual environment module, requires at least version 3.3 of Python.
+
+To create a virtual environment, make a folder to store your virtual environment (referred to below as `YOUR_VENV_PATH`). Then, use the `venv` module to set it up:
+
+```
+python3 -m venv YOUR_VENV_PATH
```

Comment:
Is `python3` available on all systems? I thought on Windows the binary was just `python` but am probably remembering wrong 😋